### ISSUE TITLE
Crashes scanning anything - all stable/beta versions above 1.4.25 x64 - Windows 10 HP

### ISSUE THREAD

**Issue by AVRider** [RandomEngy/VidCoder#98]
There was a period of time I was not using Vidcoder - not because I was using another but just not encoding. More recently, I attempted to use VC at about the intro of 1.5.32 and it crashed time after time. I scoured the web and it pointed to a possible .NET Framework problem, specifically not having 3.5 active? This is not the problem. 

BTW - Windows 10 did update during this time automatically.

I went back in time, through trial and error, and discovered that installing version 1.4.25 there were no problems. This version scans everything and operates trouble free.

The LOG is empty. The crash forces the Windows error routine looking for a fix...

Is there something in particular I should be looking at that changed after VC version 1.4.25 or the corresponding Handbrake version??? Thanks in advance.

**Comment by RandomEngy** [comment]
Well the next stable version was [1.5.28](https://github.com/RandomEngy/VidCoder/releases/tag/v1.5.28), which included all the changes from 27 beta releases. You might try looking into the Betas to find out which release between 1.5.0 and 1.5.28 broke you.

**Comment by AVRider** [comment]
Will do - thanks - will report back soon

**Comment by AVRider** [comment]
1.5.7 Beta is where the error is introduced. here is the summary of changes in 1.5.7 Beta:

Updated HandBrake core to SVN 5819.
About dialog now pulls down HandBrake version from the DLL.
Added a confirmation dialog to Stop if the encode has been going on for more than 5 minutes.

Fixed handling of unicode characters for input and output filenames. We now encode to UTF-8 before passing to HandBrake.

Fixed a crash in the queue multiple titles dialog.
Added code to rescue tool windows which get placed outside of the visible screen area.

**Comment by AVRider** [comment]
1.5.7 Beta the error was introduced. I saw something about this in another
forum related to XP but as previously stated I am running W10 HP x64. I
copied the hb.dll file from 1.5.6 Beta and sure enough 1.5.7 Beta now works.
Suggestion?

**Comment by RandomEngy** [comment]
Hmm. Sounds like something between HandBrake SVN 5714 and 5819. From scanning the HandBrake timeline it looks like they were introducing a lot of Intel QSV code, so maybe something from that is causing the crashes.

Is the latest HandBrake also crashing for you?

**Comment by AVRider** [comment]
Yes. I even loaded the 6/10 nightly build and it fails the same way.
Perhaps something in the last W10 update caused an unexpected change to the
environment? But which environment?

**Comment by RandomEngy** [comment]
Well the QSV encoding calls some system components so maybe an Intel driver got updated and broke it.

**Comment by AVRider** [comment]
I'm running AMD no Intel.

**Comment by RandomEngy** [comment]
Hmm, maybe it's the capability detection or something else entirely then. Just speculating. Anyway you can follow up with the HandBrake team to investigate further, since it reproduces there.

**Comment by AVRider** [comment]
Interestingly, the same issue was just posted over on their forum.
